test/set.jl

```diff
@@ -112,6 +112,18 @@ end
         @test order(set(uda, X=ReverseOrdered())) == (ReverseOrdered(), Unordered())
     end
 
+    @testset "issue #478" begin
```

Comment:
Maybe add a more descriptive testset name for future readers

Reply:
```suggestion
    # issue #478
    @testset "tuple dims and/or Symbol/Dim{Colon}/Colon replacement" begin
```